runelite-client/src/main/java/net/runelite/client/plugins/statusbars/StatusBarsConfig.java

```diff
@@ -104,10 +128,199 @@ default int hideAfterCombatDelay()
+		description = "The color of the prayer bar",
+		section = barColors,
+		position = 4
+	)
+	default Color getPrayerColor()
+	{
+		return new Color(50, 200, 200, 175);
+	}
+
+	@ConfigItem(
+		keyName = "prayerRestoreColor",
+		name = "Prayer restore color",
+		description = "The color of the prayer bar that displays prayer restore amount",
+		section = barColors,
+		position = 5
+	)
+	default Color getPrayerRestoreColor()
+	{
+		return new Color(57, 255, 186, 75);
+	}
+
+	@ConfigItem(
+		keyName = "activePrayerColor",
+		name = "Active Prayer bar color",
+		description = "The color of the prayer bar while prayers are being used",
+		section = barColors,
+		position = 6
+	)
+	default Color getActivePrayerColor()
+	{
+		return new Color(57, 255, 186, 225);
+	}
+
+	@ConfigItem(
+		keyName = "energyColor",
+		name = "Energy bar color",
+		description = "The color of the run energy bar",
+		section = barColors,
+		position = 7
+	)
+	default Color getEnergyColor()
+	{
+		return new Color(199, 174, 0, 220);
+	}
+
+	@ConfigItem(
+		keyName = "energyRestoreColor",
+		name = "Energy restore color",
+		description = "The color of the run energy bar that displays energy restore amount",
+		section = barColors,
+		position = 8
+	)
+	default Color getEnergyRestoreColor()
+	{
+		return new Color (199,  118, 0, 218);
+	}
+
+	@ConfigItem(
+		keyName = "staminaPotColor",
+		name = "Stamina pot color",
+		description = "The color of the run energy bar while affected by a stamina pot or Ring of Endurance",
+		section = barColors,
+		position = 9
+	)
+	default Color getStaminaPotColor()
+	{
+		return new Color(160, 124, 72, 255);
+	}
+
+	@ConfigItem(
+		keyName = "specialAttackColor",
+		name = "Special attack color",
+		description = "The color of the special attack bar",
+		section = barColors,
+		position = 10
+	)
+	default Color getSpecialAttackColor()
+	{
+		return new Color(3, 153, 0, 195);
+	}
+
+	@ConfigItem(
+		keyName = "poisonedColor",
+		name = "Poisoned color",
+		description = "The color of the health bar while poisoned",
+		section = barColors,
+		position = 11
+	)
+	default Color getPoisonedColor()
+	{
+		return new Color(0, 145, 0, 150);
+	}
+
+	@ConfigItem(
+		keyName = "envenomedColor",
+		name = "Envenomed color",
+		description = "The color of the health bar while envenomed",
+		section = barColors,
+		position = 12
+	)
+	default Color getEnvenomedColor()
+	{
+		return new Color(0, 65, 0, 150);
+	}
+
+	@ConfigItem(
+		keyName = "diseasedColor",
+		name = "Diseased color",
+		description = "The color of the health bar while diseased",
+		section = barColors,
+		position = 13
+	)
+	default Color getDiseasedColor()
+	{
+		return new Color(255, 193, 75, 181);
+	}
+
+	@ConfigItem(
+		keyName = "parasiteColor",
+		name = "Parasite color",
+		description = "The color of the health bar while afflicted with parasites",
+		section = barColors,
+		position = 14
+	)
+	default Color getParasiteColor()
+	{
+		return new Color(196, 62, 109, 181);
+	}
+
+
+
+
+
```

Comment:
Can you please remove the additional whitespace here?

Reply:
Done!